tests/integration/adapter/rest-adapter-test.js

```diff
@@ -56,7 +57,7 @@ test("find - basic payload", function(assert) {
   run(store, 'find', 'post', 1).then(assert.wait(function(post) {
     assert.equal(passedUrl, "/posts/1");
     assert.equal(passedVerb, "GET");
-    assert.equal(passedHash, undefined);
+    assert.deepEqual(passedHash.data, {});
```

Comment:
Would it be preferred to continue returning `undefined` rather than `{}` in these cases? `adapter.ajax` seems to act the same with `{ data: {} }` and `{}`.